config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -686,7 +686,7 @@ periodics:
       - --env=KUBE_FEATURE_GATES=AllAlpha=true,CSIMigration=false
       - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
       - --runtime-config=api/all=true
-      - --test_args=--ginkgo.focus=\[Feature:(Audit|BlockVolume|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes)\]|Networking --ginkgo.skip=Networking-Performance|IPv6|Feature:Volumes --minStartupPods=8
+      - --test_args=--ginkgo.focus=\[Feature:(Audit|BlockVolume|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes)\]|Networking --ginkgo.skip=Feature:(SCTPConnectivity|Volumes|Networking-Performance|IPv6) --minStartupPods=8
```

Comment:
I'm not sure if this is actually correct. Can someone confirm or point me in the right direction if it's wrong?

Reply:
the `Networking-Performance and IPv6` have the `Feature:` as well? why was not like this before?

myabe we keep: `--ginkgo.skip=Networking-Performance|IPv6|Feature:(Volumes| SCTPConnectivity)` ? not sure as well

Reply:
@cpanato This makes sense to me. I modified the PR to do what you proposed, PTAL again.